src/components/Button/Button.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from "react";
+import "./Button.css";
+
+class Button extends Component {
+  state = {
+    counterValue: 1
+  };
+
+  incrementHandler = () => {
+    let counterIncrement = this.state.counterValue;
+    counterIncrement++;
+    console.log("Spustil som incrementHandler!");
```

Comment:
It is good habit to write all comments and console.log in english (same as variable names etc) - i happens a lot that these changes are commited and then non-CZ/SK speakers don't know if this should stay here or no :)